custom-connectors/ServiceNow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3130 @@
+      "get": {
+        "summary": "Get Record Types",
+        "description": "Lists available record types",
+        "operationId": "GetRecordTypes",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GetTypesResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-visibility": "advanced"
+      }
+    }
+  },
+  "definitions": {
+    "GetTypesResponse": {
+      "type": "object",
+      "properties": {
+        "result": {
+          "description": "Result set of all record types for current instance",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "label": {
+                "description": "Display name of the record type.",
+                "type": "string",
+                "x-ms-summary": "Value"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SysIdParameter": {
+      "name": "sys_id",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-summary": "System ID",
+      "description": "System ID of the record",
+      "x-ms-visibility": "important",
+      "x-ms-url-encoding": "single"
+    },
+    "DisplaySystemRefs": {
+      "name": "sysparm_display_value",
+      "in": "query",
+      "type": "boolean",
+      "required": false,
+      "x-ms-summary": "Display System References",
+      "description": "Determines if system references should be included",
+      "x-ms-visibility": "advanced",
+      "default": false
+    },
+    "DisplayInputSystemRefs": {
+      "name": "sysparm_input_display_value",
+      "in": "query",
+      "type": "boolean",
+      "required": false,
+      "x-ms-summary": "Display Input System References",
+      "description": "Flag that indicates whether to set field values using the display value or the actual value",
+      "x-ms-visibility": "advanced",
+      "default": false
+    },
+    "ExcludeRefLinks": {
+      "name": "sysparm_exclude_reference_link",
+      "in": "query",
+      "type": "boolean",
+      "required": true,
+      "x-ms-summary": "Exclude Reference Links",
+      "description": "Determines if reference links should be excluded",
+      "x-ms-visibility": "advanced",
+      "default": true
+    },
+    "SysParmFields": {
+      "name": "sysparm_fields",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "x-ms-summary": "Fields to return in response",
+      "x-ms-visibility": "advanced",
+      "description": "Fields to return in response"
+    }
+  },
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://dev91547.service-now.com/oauth_auth.do",
+      "tokenUrl": "https://dev91547.service-now.com/oauth_token.do",
```

Comment:
Here as well.

Reply:
@ShyamSunder90 , @mamurshe  - looks like we have duplicate entries for security configuration. It seems the UX will stamp the value in the swagger file as well, while the CLI tool generates the api properties. Ideally, we would only require at one place.